samples/Jetsons/Jetsons/Jetsons.csproj

```diff
@@ -35,15 +35,4 @@
 		<ProjectReference Include="..\Microsoft.Restier.Providers.InMemory\Microsoft.Restier.Providers.InMemory.csproj" />
 	</ItemGroup>
 
-	<ItemGroup>
-		<Reference Include="Microsoft.Restier.AspNetCore">
-			<HintPath>..\RESTierAssemblies\Microsoft.Restier.AspNetCore.dll</HintPath>
-			<CopyLocal>True</CopyLocal>
-		</Reference>
-		<Reference Include="Microsoft.Restier.Core">
-			<HintPath>..\RESTierAssemblies\Microsoft.Restier.Core.dll</HintPath>
```

Comment:
Does this mean that the folder `RESTierAssemblies` and the two DLLs can be deleted?

Reply:
Yes -- good catch; thanks!